examples/kubernetes/connectivity-check/connectivity-check-single-node.yaml

```diff
@@ -268,45 +268,6 @@ spec:
 ---
 apiVersion: apps/v1
 kind: Deployment
-metadata:
```

Comment:
Is this change intentional in this commit?

Reply:
Kinda, yeah. I mean, I made modifications to the Makefile, then I re-ran `make -C examples/kubernetes/connectivity-check/ clean all` and it removed this section.

Seems to me like from the Makefile, the `single-node` variation in the connectivity check is not supposed to include any hostport checks, and yet the file as checked in has them. I'm guessing we meant to re-generate these YAMLs after an earlier commit but we never did.

I could move this blob out to a separate commit if you think that's meaningful. I didn't want to try to run the above make command before adding the `sort` because I wasn't sure that the file listing order is consistent across different machines.

Reply:
FYI I removed this commit with the new approach, and validated the diff against a deployment of the existing connectivity check. I can provide that diff when I update the PR description.